### DISCUSSION TITLE
Betweenness centrality

### DISCUSSION THREAD

**Discussion by wwwqqqlll** [community/community#36681] [Other Feature Feedback, Questions, & Ideas] [answered]
How can we use rademacher and vc-dim to compute approximate betweenness centrality with probability 1-δ，ε error

**Comment by ettaboyle** [comment, marked as answer]
Hi @wwwqqqlll,

Thanks for being a part of the GitHub Community! We're glad you're here.

I wanted to jump in and mention that if you're looking for help for this specific topic, you might want to try asking for help somewhere that focuses on [Betweenness Centrality](https://neo4j.com/docs/graph-data-science/current/algorithms/betweenness-centrality/). It's definitely possible that another GitHub user might have run into this same issue and can help, but the GitHub Community on Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!
